code/game/objects/items/weapons/circuitboards/machine_circuitboards.dm

```diff
@@ -673,39 +613,34 @@
 /obj/item/weapon/circuitboard/machine/rdserver
 	name = "R&D Server (Machine Board)"
 	build_path = /obj/machinery/r_n_d/server
-	origin_tech = "programming=3"
 	req_components = list(
 		/obj/item/stack/cable_coil = 2,
 		/obj/item/weapon/stock_parts/scanning_module = 1)
 
 /obj/item/weapon/circuitboard/machine/bsa/back
 	name = "Bluespace Artillery Generator (Machine Board)"
-	build_path = /obj/machinery/bsa/back
-	origin_tech = "engineering=2;combat=2;bluespace=2" //No freebies!
+	build_path = /obj/machinery/bsa/back //No freebies!
 	req_components = list(
 		/obj/item/weapon/stock_parts/capacitor/quadratic = 5,
 		/obj/item/stack/cable_coil = 2)
 
 /obj/item/weapon/circuitboard/machine/bsa/middle
 	name = "Bluespace Artillery Fusor (Machine Board)"
 	build_path = /obj/machinery/bsa/middle
-	origin_tech = "engineering=2;combat=2;bluespace=2"
 	req_components = list(
 		/obj/item/weapon/ore/bluespace_crystal = 20,
 		/obj/item/stack/cable_coil = 2)
 
 /obj/item/weapon/circuitboard/machine/bsa/front
 	name = "Bluespace Artillery Bore (Machine Board)"
 	build_path = /obj/machinery/bsa/front
-	origin_tech = "engineering=2;combat=2;bluespace=2"
 	req_components = list(
 		/obj/item/weapon/stock_parts/manipulator/femto = 5,
 		/obj/item/stack/cable_coil = 2)
 
 /obj/item/weapon/circuitboard/machine/dna_vault
 	name = "DNA Vault (Machine Board)"
-	build_path = /obj/machinery/dna_vault
-	origin_tech = "engineering=2;combat=2;bluespace=2" //No freebies!
+	build_path = /obj/machinery/dna_vault //No freebies!
```

Comment:
right that was a fuckup while removing origin techs i'll remove it when i'm done and get around to it.